src/compiler/compile/render-dom/wrappers/Element/index.ts

```diff
@@ -777,6 +778,7 @@ export default class ElementWrapper extends Wrapper {
 		block.builders.fix.add_block(deindent`
 			@fix_position(${this.var});
 			${stop_animation}();
+			${outro && `@prepare_outro(${this.var}, ${rect});`}
```

Comment:
Should this just call `add_transform` directly? It doesn't seem like we're gaining anything by wrapping the call with the `prepare_outro` function

```suggestion
			${outro && `@add_transform(${this.var}, ${rect});`}
```

Reply:
Sure, we can call add_transform directly, I add your suggestion. 😄 

My intention was to 1) by the method name give a clue _why_ something is happening here and 2) leave _what_ to happen to animation.ts. Also _prepare_outro_ in animation.ts documents, that animations can (and do) interfere with intros and outros. On the other hand animation.ts does not necessarily need to _know_ about outros at all, which might seem a little messy when reading.

To be honest, names like _fix_position_ say, _that_ something is happening, but not _why_. I don't know why fix_position is there at all only that it doesn't work right without it. 😄